prow/config.yaml

```diff
@@ -40,6 +40,17 @@ deck:
     announcement: "This page -- spyglass -- is the default viewer for prow.{{if .ArtifactPath}} For now, the deprecated gubernator page spyglass replaces <a href='https://gubernator.k8s.io/build/{{.ArtifactPath}}'>remains available</a>.{{end}}"
 
   tide_update_period: 1s
+  # Basing this on istio/test-infra/OWNERS for now.
+  rerun_auth_config:
+    authorized_users:
+    - chxchx
+    - cjwagner
+    - fejta
+    - hklai
+    - icygalz
+    - rkpagadala
+    - utka
+    - yutongz
```

Comment:
+ @howardjohn and @geeknoid ?

Currently we just support a list of users who are allowed to trigger any job. Soon we'll also support:
- specifying authorized users per-job in the job config
- allowing anyone who would be able to `/test` or `/retest` a presubmit from GH

Thanks to @mirandachrist for the new functionality!

Reply:
Whoops sorry for the delay on this comment Michelle. Feel free to merge as is, we can do this in a follow up.
/lgtm
/approve

Reply:
Yeah, I think in a follow up works, others can PR it as well ^^